client/rest-client/src/main/java/org/infinispan/client/rest/impl/okhttp/RestSchemasClientOkHttp.java

```diff
@@ -0,0 +1,66 @@
+package org.infinispan.client.rest.impl.okhttp;
+
+import static org.infinispan.client.rest.impl.okhttp.RestClientOkHttp.TEXT_PLAIN;
+import static org.infinispan.client.rest.impl.okhttp.RestClientOkHttp.sanitize;
+
+import java.util.concurrent.CompletionStage;
+
+import org.infinispan.client.rest.RestResponse;
+import org.infinispan.client.rest.RestSchemaClient;
+
+import okhttp3.Request;
+import okhttp3.RequestBody;
+
+/**
+ * @since 12.0
```

Comment:
is this only for the next release cycle?

Reply:
Do we have another 11.x release?

Reply:
yes, 11.0.1.Final is needed to be consumed downstream.

Reply:
ok, I will change it

Reply:
we can keep this on hold until we branch 11.x (that should happen this week)

Reply:
+1
This is a not needed in 11.x anyway